v3/integration/config.json

```diff
@@ -303,7 +303,7 @@
     },
     "e_dnsname_contains_bare_iana_suffix": {},
     "e_dnsname_empty_label": {
-      "ErrCount": 11
+      "ErrCount": 10
```

Comment:
The certificate in question here is https://crt.sh/?q=c5e0cc80ea898c2a9e2d09b0960df895535017eac46050c02458d16c6f6841d5 which failed because it's CommonName of `MONICOMP Kereskedelmi és Szolgáltató Zrt.` would get split on `.`, resulting in an "empty label" on the right hand side of the `.`.